### PR TITLE
Lesson14 user form test

### PR DESCRIPTION
**Pull request by olyastrilectka**

### CODE REVIEW

**Review by Stasevych — COMMENTED** [review]
Hello! I checked your HW! It seems like everything works correctly and your test is clear as for me. Thanks

**Comment by CyberSergo** [comment]
Hi, Olya. I liked your homework, especially the last negative test.
Advice:
- you have several repeating locators in your code, it is better to put them in variables.
- variables, like "faker" it is better to put them in a separate JS module

File: cypress/integration/lesson14.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const FAKE_ADDRESS = faker.address.secondaryAddress();
+const FAKE_PHONE = faker.phone.phoneNumber();
+
+describe('Lesson14: Text Box Form', () => {
+  before(() => {
+    cy.visit('https://demoqa.com/text-box');
+  });
+
+  afterEach(() => {
+    cy.reload(); // cleanup text from the inputs fields
```
Comment (VladyslavKryvenko) [inline cypress/integration/lesson14.spec.js]: We can use "before" commands like "cy.clearLocalStorage() and cy.clearCookies()"

**Review by VladyslavKryvenko — APPROVED** [review]
Good job, don't comment, it's redundant
10\10

**Comment by landriiuk** [comment]
U are able to merge this branch into master

File: cypress/integration/lesson14.spec.js (hunk 1 above)
Comment (olyastrilectka) [reply cypress/integration/lesson14.spec.js]: thx, I have changed

**Pull request merged**